backend/src/v2/driver/driver.go

```diff
@@ -313,6 +313,12 @@ func makePodSpecPatch(
 		return "", fmt.Errorf("failed to make podSpecPatch: %w", err)
 	}
 
+	// Convert environment variables
+	userEnvVar := make([]k8score.EnvVar, 0)
+	for _, envVar := range container.GetEnv() {
+		userEnvVar = append(userEnvVar, k8score.EnvVar{Name: envVar.GetName(), Value: envVar.GetValue()})
```

Comment:
Curious if env has a dynamic value, e.g. an output from an upstream component, what would `envVar.GetValue()` return?

Reply:
I'm not sure about this, but I don't think the function here resolves the dynamic value. It likely returns a placeholder,  because the input comes from `*pipelinespec.PipelineDeploymentConfig_PipelineContainerSpec`, which doesn't seem to have been changed.